web/client/api/searchText.js

```diff
@@ -27,12 +27,15 @@ const defaultFromTextToFilter = ({searchText, staticFilter, blacklist, item, que
     filter = filter ? filter.concat(staticFilterParsed) : staticFilterParsed || null;
     return filter;
 };
+
 let Services = {
-    nominatim: (searchText, options = {}) =>
+    nominatim: (searchText, options = {
```

Comment:
if nominatim ignores it, you don't have to set a default. You can specify otherwise, documenting this file, how the result of the promise is shaped.